### PR TITLE
Cleaned up for public consumption

### PR DESCRIPTION
**Pull request by willhaines**
So, this is a whole thing and I don't expect a full read-through right now -- we can go over the code in detail before(if) we go prod with it down the line. 

I'm really just making a PR because that's our workflow :smile:, but I'm also moving this into a public repo so a quick skim for any CU-specific stuff I might have left behind would be great.

### CODE REVIEW

File: bump_docker_base.sh [hunk 1]
```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+#
+# This script bumps the version number pinned on the FROM line of a Dockerfile
+#
+# Author: wiha1292
```
Comment (davegoldhammer) [inline bump_docker_base.sh]: Not sure if it's important, but you could make this your GitHub user in the spirit of avoiding any CU-specific references.

Comment (willhaines) [reply bump_docker_base.sh]: Good catch, thanks!

File: self_service/translate.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+                if not pol.path in policies[policy_name]:
+                    policies[policy_name][pol.path] = set()
+
+                # Add any new capabilities defined for the same target+path
+                policies[policy_name][pol.path].update(
+                    {cap.value for cap in pol.capabilities}
+                )
+
+    # keep paths that end in *, but don't contain any +
+    #sanitized_paths = { p for p in all_paths if (not '+' in p and p[-1] == '*') }
```
Comment (davegoldhammer) [inline self_service/translate.py:71]: Could these lines be removed?

Comment (willhaines) [reply self_service/translate.py:71]: Fair point, but let's leave it for now. Seems to be part of a partially-baked plan to pre-create paths that don't exist yet, which may or may not be useful.

Comment (davegoldhammer) [reply self_service/translate.py:71]: Sounds good!

**Review by davegoldhammer — APPROVED** [review]
Thanks for submitting a PR for this. Wow, there is a lot of work represented here! I mostly looked things over at a high level as you mentioned, but I was glad to get some insight on this, and the README is nice.

**Pull request merged**